app/models/child_benefit_tax_calculator.rb

```diff
@@ -46,7 +48,11 @@ def nothing_owed?
   end
 
   def has_errors?
-    errors.any? || starting_children.select { |c| c.errors.any? }.any?
+    errors.any? || starting_children_errors?
+  end
+
+  def starting_children_errors?
```

Comment:
@leenagupte: we cannot assume that `is_part_year_claim=='no'` implies that I am not going to run the validations of the children. I would expect any new validation added to the children to run in all situations.